# MetaviromePipeline

*Bioinformatics Pipeline for metavirome analysis*

![Top Langs](https://img.shields.io/badge/Linux-FCC624?style=for-the-badge&logo=linux&logoColor=black)
![Top Langs](https://img.shields.io/badge/Shell_Script-121011?style=for-the-badge&logo=gnu-bash&logoColor=white)

This pipeline was developed by the Bioinformatics Lab located in the Blood Center of RibeirÃ£o Preto. Our main goal is to classify metagenomics samples. The pipeline provides a comprehensive solution for analyzing and classifying metagenomic data, helping researchers identify and characterize microbial diversity in different types of samples.
The pipeline provides a simple but comprehensive solution for analyzing and classifying metagenomic data, helping researchers identify and characterize microbial diversity in different types of samples. We've included easy bash scripts to run and test the pipeline, as well. We encourage contributions from the community and are open to suggestions and improvements to further enhance this tool. We will continue to improve this repository with examples and updates.
From Quality Control (QC) to Taxonomic Classification

For more information about the softwares we utilize:

## Quality Control and pre-processing ðŸ“Š

[FastQC](http://www.bioinformatics.babraham.ac.uk/projects/fastqc) and [fastP](https://github.com/OpenGene/fastp).

## Mapping ðŸ§¬

[BWA](https://github.com/lh3/bwa)

## Taxonomic Classification

[Kraken2](https://github.com/DerrickWood/kraken2/blob/master/docs/MANUAL.markdown) ðŸ¦‘  </br>
[Kaiju](https://github.com/bioinformatics-centre/kaiju) ðŸ¦Ž </br>
[Diamond](https://github.com/bbuchfink/diamond/wiki) ðŸ”¹</br>
[CLARK](http://clark.cs.ucr.edu/) ðŸ‘“
